### PR TITLE
web: Add fullscreen context menu item

### PR DESCRIPTION
**Pull request by relrelb**
This PR adds a fullscreen item to the right-click context menu.
Should complete "Fullscreen button" in #4.
- [x] TODO: Retain focus on fullscreen enter/exit.

### CODE REVIEW

**Comment by relrelb** [comment]
The focus problem is solved by now and this is ready for CR.

File: web/packages/core/src/ruffle-player.js [hunk 1]
```diff
@@ -242,28 +242,67 @@ exports.RufflePlayer = class RufflePlayer extends HTMLElement {
...
+                element.style.borderTopRightRadius = "5px";
+                element.style.borderTopLeftRadius = "5px";
+            }
+            if (i === items.length - 1) {
+                element.style.borderBottomRightRadius = "5px";
+                element.style.borderBottomLeftRadius = "5px";
+            }
+            element.textContent = text;
+            element.addEventListener("click", onClick);
+            this.right_click_menu.appendChild(element);
```
Comment (Herschel) [inline web/packages/core/src/ruffle-player.js:293]: Could the styling be handled in CSS in shadow-template.js, for example by using `:first-child`, `:last-child`, etc.? `right_click_menu` could be given a class and the CSS rules could look for that class specifically.

File: web/packages/demo/www/index.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
             }
 
             #main {
-                margin: 15px;
+                margin-top: 15px;
```
Comment (Herschel) [inline web/packages/demo/www/index.html:28]: What is the purpose of this change?

Comment (relrelb) [reply web/packages/demo/www/index.html:28]: Currently the demo page doesn't fit in a whole page (for me at least), so it has a very annoying scrollbar.
This change eliminates the scrollbar, but in second thought, maybe it would be better to define `margin-bottom` on `#nav` instead.

**Comment by Herschel** [comment]
Thanks! I know @pocketonion was working on adding fullscreen methods to the RufflePlayer JS API, so I've pinged him to submit a PR, and I think we can try to rebase that PR on top of this one. (Particularly I think his PR ensures things work cross-browser, particularly on Safari which still requires a `webkit` prefix for the fullscreen API).

**Comment by relrelb** [comment]
I've just added fullscreen helper methods roughly like #1568 does, so I think we can close it now.

**Review by Dinnerbone — APPROVED** [review]
👍  from me, thank you both (@relrelb and @pocketonion) for this!

We should pull this in before my typescript PR and I'll rebase that and fix it up.

**Comment by Herschel** [comment]
Thanks, looks great! Going to merge, here's some thoughts for when this gets converted to Typescript by @Dinnerbone:

 * `isFullscreen` and `fullscreenEnabled` could be getters instead of methods
 * Not sure we need `ms` and `moz` prefix, as these are standardized now and Ruffle in general won't run in older browsers where this would be required. Safari `webkit` is the only one that's still required.

**Comment by Dinnerbone** [comment]
@Herschel I've made those changes in my PR!
